task-launcher/webpack.config.cjs

```diff
@@ -27,6 +27,7 @@ const commonConfig = {
   resolve: {
     fallback: {
       path: require.resolve('path-browserify'),
+      crypto: false,
```

Comment:
Why do we need this?

Reply:
It was giving me an issue with webpack while running this locally. Interestingly , it was only happening intermittently but once it started it was consistent. I will keep this for now but I will keep testing it further and if I find a better solution, I will go with that.